python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -35,6 +35,54 @@
 import lsst.meas.algorithms as measAlg
 import lsst.meas.algorithms.utils as maUtils
 import lsst.meas.extensions.psfex as psfex
+from lsst.pipe.base import AlgorithmError
```

Comment:
This means we need to add a `setupRequired(pipe_base)` to this package's eups table.

Reply:
Done!